python/ray/tests/test_basic.py

```diff
@@ -1991,6 +1994,11 @@ def test_custom_resources(ray_start_cluster):
     cluster.add_node(num_cpus=3, resources={"CustomResource": 1})
     ray.init(redis_address=cluster.redis_address)
 
+    client_info = ray.tests.utils.get_available_client_info_from_redis({
+        "CustomResource": 1
+    })
+    available_plasma_socket_name = client_info["ObjectStoreSocketName"]
```

Comment:
This would be simpler.
```
node = cluster.add_node(num_cpus=3, resources={"CustomResource": 1})
available_plasma_socket_name = node.plasma_store_socket_name
```

Reply:
@raulchen Thanks! That is much better!